docker_interop/static-nodes.json

```diff
@@ -0,0 +1,8 @@
+[
+    "enode://b81c7d5207f96ee4239ba7597798f1e1ec6db494ead9aac0a55836b75bf8580cde01c577e406de12c057de23411e5a9fc723ad6dcd2e75262cf7d04c5613b9c7@172.16.239.16:30303", 
```

Comment:
Including itself in the static-nodes doesn't work well so well for Besu. It seems we try to obtain a peer connection to ourselves. So it would be good to remove that from the besu static nodes. Or just use peer discovery.